src/news/release-5.0.md

```diff
@@ -0,0 +1,94 @@
+- Enable dark titlebar on Windows 10 by @DavidoTek in PolyMC/PolyMC#982
+- Preliminary work towards ATLauncher pack updates by @jamierocks in PolyMC/PolyMC#897
+- Trash instances instead of deleting by @ryanccn in PolyMC/PolyMC#549
+- [Windows] Find Java installs from PATH by @TayouVR in PolyMC/PolyMC#1068
+
+### Changed
+
+- Allow double clicking to mark mods for download by @ErogigGit in PolyMC/PolyMC#1165
+- Improve language model by @Scrumplex in PolyMC/PolyMC#1164
+- Improve loading speed of the application by @flowln in PolyMC/PolyMC#878
+- Merge Launch Buttons by @TayouVR in PolyMC/PolyMC#1190
+- Support new Prism Launcher data path by @Scrumplex, @flowln and @FayneAldan in #25 #55 #82 #88
+- Reduce code duplication in tasks, fix some bugs and add some tests by @flowln in PolyMC/PolyMC#966
+- Refactor resource models by @flowln in PolyMC/PolyMC#1052 PolyMC/PolyMC#1094
+- Restructure Instance sidebar by @DioEgizio in #33
+- Restructure tests by @Scrumplex in PolyMC/PolyMC#1133
+- Switch to new API keys by @Scrumplex in #23
+- Switch to new branding by @Scrumplex, @NotNite in #13 #45 #54 #60 #92 #110
+- Switch to new versioning scheme by @Scrumplex in PolyMC/PolyMC#992
+- Switch to QCommandLineParser by @Scrumplex in PolyMC/PolyMC#1167
+- Switch to QSharedPointer by @flowln in PolyMC/PolyMC#1023
+- Switch to std::filesystem where possible by @flowln in PolyMC/PolyMC#1142
+- UI / UX improvements to the mod downloader by @flowln in PolyMC/PolyMC#939
+
+### Fixed
+
+- Add missing includes to fix Qt 6.4 build by @Scrumplex in PolyMC/PolyMC#1172
+- Allow adding MSA account on first start by @Scrumplex in PolyMC/PolyMC#1183
+- Allow starting rd- versions by @Scrumplex in PolyMC/PolyMC#1146
+- Fix aborting network jobs by @flowln in PolyMC/PolyMC#1123
+- Fix and improve FTB modpack installation by @flowln in #27
+- Fix hosted weblate translation website URL by @nightsnowlinouo in #68
+- Fix instance copy on Linux and macOS by @flowln in #25
+- Fix mod icon issues and change delegate for Modrinth Modpacks by @flowln in PolyMC/PolyMC#1163
+- Fix version filter in mod downloader by @flowln in PolyMC/PolyMC#997
+- Remove opted out versions from the download selector when using the 'Any' filter by @flowln in PolyMC/PolyMC#1194
+- Update libnbtplusplus submodule's URL by @flowln and @Scrumplex in #48
+- Update licensing information by @DioEgizio in #36
+- Update login message box by @Ozynt in PolyMC/PolyMC#1175
+- Use system gamemode client if possible by @Scrumplex in PolyMC/PolyMC#1210
+
+### Removed
+
+- Remove outdated information about "upcoming" mod updater by @DioEgizio in PolyMC/PolyMC#1185
+- Remove unused libraries by @DioEgizio in PolyMC/PolyMC#1208
+
+Full Changelog: <https://github.com/PrismLauncher/PrismLauncher/compare/1.4.2...5.0>
+
+You can [grab the latest download here](https://github.com/PolyMC/PolyMC/releases/tag/5.0) for your respective platform.
```

Comment:
PolyMC URL, should be changed to Prism Launcher URL.

Reply:
oh right, thanks